packages/flutter/lib/src/painting/text_style.dart

```diff
@@ -334,6 +334,15 @@ const double _kDefaultFontSize = 14.0;
 /// ```
 /// {@end-tool}
 ///
+/// #### Supported font formats
+///
+/// Font formats currently supported by Flutter :
```

Comment:
Add newline after this line.

Reply:
@GaryQian I have committed the proposed changes. Please check if there are any further changes to be made.

Thank you.

Reply:
It LGTM now, thank you for your contribution!

Reply:
@GaryQian the cookbook doesn't mention .ttc too. Should it also be updated?